modnet/fit_genetic.py

```diff
@@ -19,25 +23,77 @@ def __init__(
         self,
         data: MODData,
         size_pop=15,
-        num_epochs=5,
-        prob_mut=0.8
+        num_epochs=5
         ):
 
         """Initializes parameters used in this class.
 
         Parameters:
             size_pop: Size of the population.
             num_epochs: Number of generations.
-            prob_mut: Probability the mutation occurs.
             data: A 'MODData' that has been featurized and feature selected.
         """
 
         self.size_pop = size_pop
         self.num_epochs = num_epochs
-        self.prob_mut = prob_mut
         self.data = data
 
 
+    def shuffle_MD(
+        self,
+        data: MODData,
+        random_state: int
+        ):
+
+        """Shuffles the MODData data.
+        
+        Parameters:
+            data: A 'MODData' that has been featurized and feature selected.
+            random_state: It affects the ordering of the indices, which controls the randomness of each fold.
+        """
+
+        data = copy.deepcopy(data)
+        ids = data.df_targets.sample(frac=1,random_state=random_state).index
+        data.df_featurized = data.df_featurized.loc[ids]
+        data.df_targets = data.df_targets.loc[ids]
+        data.df_structure = data.df_structure.loc[ids]
+    
+        return data
+
+
+    def MDKsplit(
```

Comment:
We should probably define these utility functions somewhere else.

Reply:
Yes that would be more convenient